src/arguments.js

```diff
@@ -56,9 +56,12 @@ export async function getCommandArguments() {
             .env(ENV_VAR.SMTP_USERNAME))
         .addOption(new Option('--smtppassword <password>', 'smtp password')
             .env(ENV_VAR.SMTP_PASSWORD))
-        .addOption(new Option('--subject <subject>', 'email Subject')
+        .addOption(new Option('--subject <subject>', 'email subject')
             .default(DEFAULT_EMAIL_SUBJECT)
             .env(ENV_VAR.EMAIL_SUBJECT))
+        .addOption(new Option('--note <subject>', 'email note')
```

Comment:
does `email note` mean email body?

Reply:
No, It's the note section in email body
<img width="392" alt="image" src="https://user-images.githubusercontent.com/56703525/214383303-406be59f-914f-4afd-af0f-52a2455d0d59.png">

Reply:
would users know what it means? i've not heard email note before

Reply:
does email body or message makes more sense? any other suggestion?